modules/ROOT/pages/_partials/rn-new-and-changed-ref.adoc

```diff
@@ -1,52 +1,15 @@
 [id='rn-new-and-changed-ref']
 = New and changed features
 
-* Support for TLS 1.0 and TLS 1.1 is removed from {PRODUCT}. Only TLS 1.2 is supported in this release.
+* Any {PRODUCT_SHORT} user can now determine which {walkthrough-name}s are available in a cluster. 
 
 * {code-ready-name-short} is upgraded to {code-ready-version}. Existing workspaces are not compatible with this version of {code-ready-name-short}.
 +
 Red Hat recommends that you recreate any existing workspaces using {code-ready-name-short} {code-ready-version}.
 
-* The following Solution Patterns are not available in this release:
+* You can now configure AMQ Online as described in xref:configuring-amq[].
 +
 --
-** _Protecting Applications using Red Hat Single Sign-On (SSO)_.
-** _Low-code API development and SaaS integration_.
---
-+
-Ignore references to these Solution Patterns in the documentation.  
-
-* You can now configure AMQ Online as described in xref:configuring-amq[].
-
-
-== New component versions
-
-This release includes new versions of the following components:
-
-[cols="h,"]
-|===
-
-|Component
-|*Release Notes*
-
-|{fuse-name}
-|link:{fuse-rn-docs}[{fuse-version}]
-
-|{amq-online-name}
-|link:{amq-rn-docs}[{amq-online-version}]
-
-|{three-scale-name}
-|link:{three-scale-rn-docs}[{three-scale-version}]
-
-|{code-ready-name}
-|link:{code-ready-rn-docs}[{code-ready-version}]
-
-|===
-
-
-[id='configuring-amq']
```

Comment:
if you remove this, you break the link to it @briandooley

Reply:
The link is no longer necessary. The text of the section is now included in the list of new and changed features and is no longer its own section.

![image](https://user-images.githubusercontent.com/3627775/81062808-8db3cf00-8ece-11ea-81f4-b4c33d0c9739.png)

Reply:
References to that should have been taken out as that section is now included in the new and changed features.  Have created [this](https://github.com/integr8ly/user-documentation/pull/131) to fix.